README.md

```diff
@@ -32,52 +32,31 @@ support AMD CPU, ARM CPU, and NVidia GPU through ONNX Runtime with limited testi
 * [2024/07] Performance optimizations and usability improvements on [client-side](./docs/source/3x/client_quant.md).
 
 ## Installation
+Choose the necessary framework dependencies to install based on your deploy environment.
 ### Install Framework
-#### Install torch for CPU
-```Shell
-pip install torch --index-url https://download.pytorch.org/whl/cpu
+[Install intel_extension_for_pytorch for CPU](https://intel.github.io/intel-extension-for-pytorch/cpu/latest/)
```

Comment:
Need to split those into several lines, make it clearer. 
![image](https://github.com/user-attachments/assets/917e971e-dd61-49f6-8a20-cd3745c7f17d)

Reply:
- [Install intel_extension_for_pytorch for CPU](https://intel.github.io/intel-extension-for-pytorch/cpu/latest/) 
- [Install intel_extension_for_pytorch for XPU](https://intel.github.io/intel-extension-for-pytorch/xpu/latest/) 
- [Use Docker Image with torch installed for HPU](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html#bare-metal-fresh-os-single-click)
Note: There is a version mapping between Intel Neural Compressor and Gaudi Software Stack, please refer to this [table](https://github.com/intel/neural-compressor/blob/e0b65444535a3ce502b2cf5c65921dfa29786dd0/docs/source/3x/gaudi_version_map.md) and make sure to use a matched combination.
- [Install torch for other platform](https://pytorch.org/get-started/locally)
- [Install tensorflow](https://www.tensorflow.org/install)